app/models/container_image.rb

```diff
@@ -16,6 +17,10 @@ def full_name
     result
   end
 
+  def display_registry
+    container_image_registry.present? ? container_image_registry.full_name : "Unknown image source"
+  end
```

Comment:
can this be somehow unified with the similar display on an image's summary page? (just if it's possible, to avoid duplicate message in 2 places)